components/Header/Header.js

```diff
@@ -0,0 +1,3 @@
+export default function Header() {
+  return <h1>Task Flow</h1>;
+}
```

Comment:
Technically, you do not need a component here yet as your component only renders a single element. 🙂

Reply:
We are thinking about changing this in the future by adding a logo etc. 😊